app/src/main/java/org/mozilla/fenix/search/SearchFragment.kt

```diff
@@ -188,19 +183,6 @@ class SearchFragment : Fragment(), BackHandler {
 
         view.toolbar_wrapper.clipToOutline = false
 
-        searchShortcutsButton.setOnClickListener {
-            val isOpen = searchStore.state.showShortcutEnginePicker
-            searchStore.dispatch(SearchFragmentAction.ShowSearchShortcutEnginePicker(!isOpen))
-
-            if (isOpen) {
-                requireComponents.analytics.metrics.track(Event.SearchShortcutMenuClosed)
```

Comment:
Can we remove these telemetry items? (Should not using these events break the build on telemetry?)